docs/src/modules/utils/helpers.ts

```diff
@@ -170,6 +170,7 @@ export function getDependencies(
     '@mui/private-theming': getMuiPackageVersion('theming', muiCommitRef),
     '@mui/core': getMuiPackageVersion('core', muiCommitRef),
     '@mui/utils': getMuiPackageVersion('utils', muiCommitRef),
+    '@mui/material-next': getMuiPackageVersion('material-next', muiCommitRef),
```

Comment:
Are we going to have a `latest` version of this package when we release v5 (probably yes, but just to make sure)? I am asking because when we do the stable release it would default to `latest` if there is no commit ref.